### ISSUE TITLE
Android 11 SVG rendering blank

### ISSUE THREAD

**Issue by vtkachen111** [molinch/FFImageLoading#1526]
## 🐛 Bug Report

I updated my Samsung s10+ to android 11 and in my app all the svgs stopped rendering

### Configuration

**Version:** 1.x

**Platform:** 
- [ ] :iphone: iOS
- [X ] :robot: Android
- [ ] :checkered_flag: WPF
- [ ] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [ ] :monkey: Xamarin.Forms

**Comment by sarkarstanmoy** [comment]
same issue

**Comment by jaredballen** [comment]
I have reports of the same issue in my App. However, it appears that only images that are not within the main Apps assembly that are not loading. Most of my assets are another binary. It appears those are not displaying; but, my local ones are. I'm still trying to confirm. So far it's only be reported on Samsung S10, S10+ and S20 that are running Android 11. I have none of these devices to test with. I will update the case when I have confirmed this observation.

**Comment by vtkachen111** [comment]
Workaround that fixed this for me... install skiasharp on your android project

**Comment by jaredballen** [comment]
Interesting. I already include SkiaSharp (v2.80.2) and SkiaSharp.Forms (v2.80.2) in my Android project and I am having this issue.

**Comment by jaredballen** [comment]
Forget my last comment. @vtkachen111 it looks like your workaround worked for me as well. Upon closer inspection I needed to consolidate the SkiaSharp versions in my Android project. Now that they are all at the same level, I've been able to have someone confirm the images are now showing on an S10.

**Comment by vtkachen111** [comment]
Yeah lol, freaked me out a little bit when my phone updated to android 11 and opening my app didn't show any buttons i've made out of svgs. I don't think this library is maintained much anymore... hopefully it'll re-appear as part of community toolkit or xamarin essentials

**Comment by sarkarstanmoy** [comment]
Worked for me also . Thanks

**Comment by progkhalifa** [comment]
for android 11
 Call Init() Static Method in SvgCachedImage Class before using new object from SvgCachedImage()

 Like that
  FFImageLoading.Svg.Forms.SvgCachedImage.Init();
  var svgCachedImage = new FFImageLoading.Svg.Forms.SvgCachedImage
           {
                ReplaceStringMap = xxxxxxxxxxx ,
                Source = "xxxxxxxxxxx"
           };

**Comment by WongWeiLoon** [comment]
Great, update SkiaSharp.Views.Forms package to v2.80.3 do the tricks 
Thanks a lot

**Comment by LuisiitoDev** [comment]
Hi guys, any update about this issue?,  I've got the same😅

**Comment by awalker-dsg** [comment]
We saw the blank SVG issue on a Samsung Galaxy S10 running Android 11. 

Our app uses the Xamarin.FFImageLoading.Svg.Forms Nuget, which ultimately uses SkiaSharp. However, we did not explicitly include SkiaSharp in our Visual Studio project. Adding an explicit reference to the latest (2.80.3) version of SkiaSharp to our project's Nugets fixed the problem for us.

**Comment by kofanov** [comment]
Thank you very much!
Simple adding SkiaSharp library helped to me!

**Comment by alexmartinezm** [comment]
> We saw the blank SVG issue on a Samsung Galaxy S10 running Android 11.
> 
> Our app uses the Xamarin.FFImageLoading.Svg.Forms Nuget, which ultimately uses SkiaSharp. However, we did not explicitly include SkiaSharp in our Visual Studio project. Adding an explicit reference to the latest (2.80.3) version of SkiaSharp to our project's Nugets fixed the problem for us.

Worked like a charm

**Comment by BillyMartin1964** [comment]
Still not fixed. No embedded svg shows up in android release.

**Comment by MarioG108** [comment]
> We saw the blank SVG issue on a Samsung Galaxy S10 running Android 11.
> 
> Our app uses the Xamarin.FFImageLoading.Svg.Forms Nuget, which ultimately uses SkiaSharp. However, we did not explicitly include SkiaSharp in our Visual Studio project. Adding an explicit reference to the latest (2.80.3) version of SkiaSharp to our project's Nugets fixed the problem for us.

Still not fixed. No embedded SVG shows up in android 11. 

**Solution**:  **[Installs the NuGet package SkiaSharp version (2.80.3) or bigger.](https://www.nuget.org/packages/SkiaSharp/2.88.0)**  (I installed it in my Android project only and is working like a charm) .

**Comment by bricefriha** [comment]
> > We saw the blank SVG issue on a Samsung Galaxy S10 running Android 11.
> > Our app uses the Xamarin.FFImageLoading.Svg.Forms Nuget, which ultimately uses SkiaSharp. However, we did not explicitly include SkiaSharp in our Visual Studio project. Adding an explicit reference to the latest (2.80.3) version of SkiaSharp to our project's Nugets fixed the problem for us.
> 
> Still not fixed. No embedded SVG shows up in android 11.
> 
> **Solution**: **[Installs the NuGet package SkiaSharp version (2.80.3) or bigger.](https://www.nuget.org/packages/SkiaSharp/2.88.0)** (I installed it in my Android project only and is working like a charm).

Thanks so much, would it be possible to fix this through a PR? 🙂

**Comment by devperson** [comment]
Updating SkiaSharp to the latest version(2.88.0) fixed the issue for me

**Comment by Kenny-Ali** [comment]
Hello guys, 

I have installed Skiasharp on my Xamarin Android project as you recommended but don't know what else to do? Do I need to substitute FFimageLoading with this Skiasharp or what? I'm losing users like crazy due to this.

I have this:

ImageService.Instance.LoadUrl(URLImage404)
                        .Retry(3, 250)
                        .LoadingPlaceholder("LoadingImage", FFImageLoading.Work.ImageSource.CompiledResource)
                        .ErrorPlaceholder("LoadingImage", FFImageLoading.Work.ImageSource.CompiledResource)
                        .Into(cardview.imgIcon);

**Comment by Kenny-Ali** [comment]
> > We saw the blank SVG issue on a Samsung Galaxy S10 running Android 11.
> > Our app uses the Xamarin.FFImageLoading.Svg.Forms Nuget, which ultimately uses SkiaSharp. However, we did not explicitly include SkiaSharp in our Visual Studio project. Adding an explicit reference to the latest (2.80.3) version of SkiaSharp to our project's Nugets fixed the problem for us.
> 
> Still not fixed. No embedded SVG shows up in android 11.
> 
> **Solution**: **[Installs the NuGet package SkiaSharp version (2.80.3) or bigger.](https://www.nuget.org/packages/SkiaSharp/2.88.0)** (I installed it in my Android project only and is working like a charm) .

Just need to install it? What code do you put after?

**Comment by Kenny-Ali** [comment]
> Updating SkiaSharp to the latest version(2.88.0) fixed the issue for me

Just need to install it? What code do you put after? Were you using FFImageLoading as well?

**Comment by Kenny-Ali** [comment]
> Forget my last comment. @vtkachen111 it looks like your workaround worked for me as well. Upon closer inspection I needed to consolidate the SkiaSharp versions in my Android project. Now that they are all at the same level, I've been able to have someone confirm the images are now showing on an S10.

Consolidate how?

**Comment by marcoaurelyo** [comment]
> > We saw the blank SVG issue on a Samsung Galaxy S10 running Android 11.
> > Our app uses the Xamarin.FFImageLoading.Svg.Forms Nuget, which ultimately uses SkiaSharp. However, we did not explicitly include SkiaSharp in our Visual Studio project. Adding an explicit reference to the latest (2.80.3) version of SkiaSharp to our project's Nugets fixed the problem for us.
> 
> Still not fixed. No embedded SVG shows up in android 11.
> 
> **Solution**: **[Installs the NuGet package SkiaSharp version (2.80.3) or bigger.](https://www.nuget.org/packages/SkiaSharp/2.88.0)** (I installed it in my Android project only and is working like a charm) .

Success.
